src/status_im/contexts/settings/wallet/events_test.cljs

```diff
@@ -1,20 +1,81 @@
 (ns status-im.contexts.settings.wallet.events-test
   (:require
-    [cljs.test :refer-macros [deftest is]]
+    [cljs.test :refer-macros [deftest is testing]]
     matcher-combinators.test
     [status-im.contexts.settings.wallet.events :as sut]))
 
-(def key-uid "0xfef454bb492ee4677594f8e05921c84f336fa811deb99b8d922477cc87a38b98")
+(def mock-key-uid "key-1")
+(defn mock-db
+  [keypairs accounts]
+  {:wallet  {:keypairs keypairs
+             :accounts accounts}
+   :profile {:profile {:key-uid "test-key-uid"}}})
 
-(deftest rename-keypair-test
+(deftest test-rename-keypair
   (let [new-keypair-name "key pair new"
         cofx             {:db {}}
         expected         {:fx [[:json-rpc/call
                                 [{:method     "accounts_updateKeypairName"
-                                  :params     [key-uid new-keypair-name]
-                                  :on-success [:wallet/rename-keypair-success key-uid new-keypair-name]
+                                  :params     [mock-key-uid new-keypair-name]
+                                  :on-success [:wallet/rename-keypair-success mock-key-uid
+                                               new-keypair-name]
                                   :on-error   fn?}]]]}]
     (is (match? expected
                 (sut/rename-keypair cofx
-                                    [{:key-uid      key-uid
+                                    [{:key-uid      mock-key-uid
                                       :keypair-name new-keypair-name}])))))
+
+(deftest test-get-keypair-export-connection
+  (let [db              (mock-db [] {})
+        sha3-pwd        "test-password"
+        keypair-key-uid "test-keypair-uid"
+        callback        (fn [connect-string] (println "callback" connect-string))]
+    (testing "test-get-keypair-export-connection"
+      (let [effects (sut/get-keypair-export-connection
+                     {:db db}
+                     [{:sha3-pwd sha3-pwd :keypair-key-uid keypair-key-uid :callback callback}])
+            fx      (:fx effects)]
+        (is (some? fx))))))
```

Comment:
Not a blocker for me, but I'm curious to hear some thoughts about how to test these effect data structures without making the tests too brittle. Should we check for the effect name or params?